src/proxy.ts

```diff
@@ -12,7 +12,9 @@ export function initializeProxy(): void {
 
     if (MAJOR_NODEJS_VERSION >= 10) {
       // `global-agent` works with Node.js v10 and above.
-      require('global-agent').bootstrap();
+      require('global-agent').bootstrap({
+        environmentVariableNamespace: '',
```

Comment:
I don't have much context, but what's the downside of using the default `GLOBAL_AGENT_` namespace rather than the blank one here? Is there an upstream dependency that depends on having `HTTP_PROXY` as the environment variable specifically?

Reply:
If you using the default `GLOBAL_AGENT` namespace, the following problems will occur:

1. This is not described in https://github.com/electron/get#proxies. Leading many to think it is `HTTP(S)_PROXY` rather than `GLOBAL_AGENT_HTTP(S)_PROXY`.  This is the main reason, as most don't actually bother to read the documentation carefully. This is especially true when this is a dependency of some software. For example `electron-fiddle`. This is why, in electron issues, you will find some Chinese who cannot provide a demo because they cannot configure the proxy properly, and some "undownloadable" issues in `electron-fiddle`
2. When the nodejs version is less than 10, xxx will be used and it will use HTTP(S)_PROXY as a proxy. And we should keep our behaviour uniform

Reply:
@BlackHole1 Since this would break existing functionality for folks who use the `GLOBAL_AGENT_` prefix, I think this would count as a major version breaking change.

If we are doing breaking changes... I wonder if we should just drop Node 10 support going forward since it's already at EOL and document the `global-agent` prefix in the README.

That would also remove our dependency to `global-tunnel-ng` :)

Reply:
Hey @erickzhao . Now no breaking changes. request re-reivew